routes/web-admin.php

```diff
@@ -10,17 +10,18 @@
 
 
 Route::prefix("admin")->group(function(){
```

Comment:
Remove this and add ->prefix('admin') to your admin routes' definition in RouteServiceProvider